src/lightning/pytorch/loops/fit_loop.py

```diff
@@ -260,17 +270,18 @@ def setup_data(self) -> None:
 
         self._data_fetcher = _select_data_fetcher(trainer, RunningStage.TRAINING)
         self._data_fetcher.setup(combined_loader)
-        iter(self._data_fetcher)  # creates the iterator inside the fetcher
+        iter(self._data_fetcher)  # Creates the iterator inside the fetcher
         max_batches = sized_len(combined_loader)
         self.max_batches = max_batches if max_batches is not None else float("inf")
         has_len_all_ranks_ = has_len_all_ranks(combined_loader, trainer.strategy, allow_zero_length)
 
         if self.max_batches == 0:
-            return
+            return  # No batches to process
 
-        # store epoch of dataloader reset for reload_dataloaders_every_n_epochs
+        # Store epoch of dataloader reset for reload_dataloaders_every_n_epochs
         self._last_train_dl_reload_epoch = trainer.current_epoch
 
+        # Validation check interval logic
```

Comment:
A comment is only useful it if sheds more light on the underlying mechanisms.
Adding "Validation check interval logic" on top of `if isinstance(trainer.val_check_interval, int) ...` is pretty redundant.

What does this logic do exactly is what will make the comment useful for readers. Conversely it's line noise.